modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -9580,10 +9580,11 @@
                                       "pattern": "^[a-z,A-Z]{2}$",
                                       "example": "OR"
                                     },
-                                    "country": {
-                                      "description": "Country of the address. USA is the value to use for United States based addresses, US will not work.",
+                                    "countryCode": {
+                                      "description": "Country code of the address.",
                                       "type": "string",
-                                      "example": "USA"
+                                      "example": "US",
+                                      "pattern": "^[a-z,A-Z]{2,6}$"
```

Comment:
This regex won't allow for the same characters in the `ClaimsApi::BRD::COUNTRY_CODES` keys. Commas don't exist in the keys, but both dashes and numbers do ("TR-01" is an example with both). `^[A-Z0-9-]{2,6}$` seems to cover everything the keys have while still being relatively simple.